### ISSUE TITLE
[Feature]: Add to chip content: Name + State

### ISSUE THREAD

**Issue by saper-2** [piitaya/lovelace-mushroom#849]
### Is your feature request related to a problem?

I wanted to display battery levels from blinds on chips but either it can only display value or name, so having 2 chips side-by-side each other with value <kbd>:battery: 45%</kbd>  <kbd>:battery: 32%</kbd> is not really "intuitive" (I will know since I created those chips, but others family members not really will know what those refer to)...

### Describe the solution you'd like

So basically I'd like to display in chip a Name and entity state glued together. e.g.: 
- Name=```Left batt.``` , 
- entity=```sensor.cover_battery_precent_1``` (e.g. value ```45%```) , 

This would display on chip:  
<kbd>:battery: Blind left 45%</kbd>

**Comment by piitaya** [comment]
You can do it with template chip
